src/content/docs/en/guides/assets.mdx

```diff
@@ -145,19 +145,57 @@ To avoid errors in your project, complete the following steps:
 
 ### Update content collections schemas
 
-A new `image` helper for content collections can validate the metadata of images located in the `src/assets/` folder, such as author photos or blog cover images.
+You can now declare images in your frontmatter as their paths relative to the `src/assets/` folder. The image will be imported and transformed into metadata that matches the signature of imported images, allowing you to pass its properties directly to the `<Image />` component or the `getImage()` function.
 
-The image metadata will be transformed to match the signature of imported images, and therefore can be passed directly to the `<Image />` component or the `getImage()` function.
+A new `image` helper for content collections lets you validate the image metadata using Zod.
 
-```ts
-import { image, defineCollection, z } from "astro:content";
+```ts title="src/content/config.ts"
+import { defineCollection, z, image } from "astro:content";
+import type { ImageMetadata } from "astro/dist/assets/types";
```

Comment:
Err, this type being importable from there is great, but not reliable. I wouldn't necessarily suggest people to import it at the moment. I'll update the exports to export something better on Monday

Reply:
Will wait on this, as without it the code yells at you!